src/main/resources/templates/fragments/piwik.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html xmlns:th="http://www.thymeleaf.org">
+<head>
+  <title>Piwik</title>
+</head>
+<body>
+<div th:fragment="piwik">
+  <script th:inline="javascript">
+                (function() {
+                    bindPiwikListener('web', /*[[${@environment.getProperty('piwik.url')}]]*/, /*[[${@environment.getProperty('piwik.siteId')}]]*/);
+                })();
```

Comment:
Is there a reason the function call arguments are commented out here? This isn't valid JavaScript: `Uncaught SyntaxError: Unexpected token `.